lib/mutant.rb

```diff
@@ -200,7 +200,14 @@ class Config
       fail_fast:         false,
       includes:          EMPTY_ARRAY,
       integration:       Integration::Null,
-      isolation:         Mutant::Isolation::Fork,
+      isolation:         Mutant::Isolation::Fork.new(
+        devnull: ->(&block) { File.open(File::NULL, File::WRONLY, &block) },
```

Comment:
Likely, but as you cannot seek the `/dev/null` it would not make a difference.

But as: `File::APPEND` is the more "strict" version of `File::WRONLY` I think append is the better choice overall. If it works I'll use it. Good catch.